src/parser.rs

```diff
@@ -865,11 +873,11 @@ mod tests {
         assert!(result.functions.contains_key("MainProcessSingleton"));
         let func = result.functions.get("MainProcessSingleton").unwrap();
         assert_eq!(func.start, 15);
-        assert_eq!(func.executed, true);
+        assert!(func.executed);
```

Comment:
We'll lose the benefit of using `pretty_assertions` here, but do we care ?? @marco-c what's your opinion ?

Reply:
I guess they're very few cases, so we can live with it if it's what clippy wants

Reply:
Alternatively I can simply turn of that specific clippy lint if that's more preferable.